packages/mdc-auto-init/index.ts

```diff
@@ -78,7 +97,8 @@ export default function mdcAutoInit(root = document, warn = CONSOLE_WARN) {
   return components;
 }
 
-mdcAutoInit.register = function(componentName, Ctor, warn = CONSOLE_WARN) {
+// tslint:disable-next-line:variable-name
```

Comment:
Which name requires this comment?

Reply:
`Ctor`. I renamed it to `Constructor` and added a comment explaining why we're disabling this tslint rule.